src/components/interface/Table/Table.js

```diff
@@ -136,7 +136,10 @@ const Table = ({
             .map((row) => (
               <tr key={getRowKey(row)}>
                 {columns.map((column) => (
-                  <td key={column.name}>
+                  <td
+                    key={column.name}
+                    className={column.cellClassRender ? column.cellClassRender(row) : null}
```

Comment:
Can we keep undefined ? to be consistend with the rest ? 
or laybe this should work ?
```suggestion
                    className={column.cellClassRender && column.cellClassRender(row)}
```

Reply:
It works perfectly so i did the update.

Reply:
just add the `cellClassRender`in the TS declaration and we'll be good :)

Reply:
I would like to, but i didn't find the TS declaration for columns... I only have:

...
export interface TableProps {
    columns: Object[];
    data: any[];
...

in Table.d.ts...

Reply:
ok :/